### ISSUE TITLE
Desktop Build...

### ISSUE THREAD

**Issue by bitmodeler** [hhanh00/zwallet#168]
### Subject of the issue
The instructions under [**Desktop Builds**](https://github.com/hhanh00/zwallet/blob/main/BUILD.md#desktop-builds) for the Rust compile are a little sketchy.

"First compile the rust code. In native/zcash-sync, edit Cargo.toml to change the library crate type to cdylib. Then cargo build --release --features=dart_ffi"

### Your environment
* version of YWallet: v1.8.1+2
* Debian Linux, Bookworm/12

### Steps to reproduce
go to {_project_dir_}/native (I guess? missing instruction...)
```
cargo build --release --features=dart_ffi
```
I get "failed to read `{_project_dir_}/native/zcash-params/Cargo.toml`"

So should I copy {_project_dir_}/Cargo.toml down to that directory??

And if I copy the Cargo.toml from the project_dir to the various native subfolders, I get:

```
cargo build --release --features=dart_ffi
error: multiple workspace roots found in the same workspace:
  /home/developer/ywallet/native/zcash-params
  /home/developer/ywallet/native/zcash-sync
  /home/developer/ywallet/native/zcash-vote
  /home/developer/ywallet
```

### Expected behaviour
No errors from rust compilation

### Actual behaviour
Error above

**Comment by bitmodeler** [comment]
??

**Comment by hhanh00** [comment]
Builds are made via Github workflows

For Linux, it looks like
```
export $(cat build.env | xargs)
PATH=$PATH:$PWD/flutter/bin
./build-scripts/build-linux.sh $FLUTTER_VERSION
```

**Comment by bitmodeler** [comment]
Thanks much! Maybe the doc can be updated; until then, I guess people can find this issue...

**Comment by hhanh00** [comment]
Closed by #169
